source/features/clean-conversation-headers.tsx

```diff
@@ -23,29 +23,41 @@ async function cleanIssueHeader(byline: HTMLElement): Promise<void> {
 async function cleanPrHeader(byline: HTMLElement): Promise<void> {
 	byline.classList.add('rgh-clean-conversation-headers');
 
+	const authorSelector = [
+		'.TimelineItem .author',
+		'.Timeline-Item [data-testid="author-avatar"] a:not([data-testid="github-avatar"])',
+	].join(',');
+
 	// Extra author name is only shown on `isPRConversation`
 	// Hide if it's the same as the opener (always) or merger
 	const shouldHideAuthor
 		= pageDetect.isPRConversation()
 		&& !byline.closest('.gh-header-sticky') // #7802
-		&& $('.author', byline).textContent === (await elementReady('.TimelineItem .author'))!.textContent;
+		&& $([
+			'.author',
+			'a[data-hovercard-url]',
+		], byline).textContent === (await elementReady(authorSelector))!.textContent;
 
 	if (shouldHideAuthor) {
 		byline.classList.add('rgh-clean-conversation-headers-hide-author');
 	}
 
-	const base = $('.commit-ref', byline);
+	const base = $([
+		'.commit-ref',
+		'[class^="BranchName"]',
+	], byline);
 	const baseBranchDropdown = $optional('.commit-ref-dropdown', byline);
 
 	// Shows on PRs: main [←] feature
 	const arrowIcon = <ArrowLeftIcon className="v-align-middle mx-1" />;
 	if (baseBranchDropdown) {
 		baseBranchDropdown.after(<span>{arrowIcon}</span>); // #5598
 	} else {
-		base.nextElementSibling!.replaceChildren(arrowIcon);
+		base.after(<span>{arrowIcon}</span>);
 	}
 
-	const baseBranch = base.title.split(':')[1];
+	const baseBranchContent = base.textContent.split(':');
```

Comment:
This change needs to be reviewed because this part of the UI is rather tricky, especially when introducing long names, escaping, cross-repo PRs, etc. However we already have a function that gets the base and head branch information, so we should probably use that one (as well as ensure that it works as expected on this new view, so that all remaining features benefit from it)

Reply:
I'm using `parseReferenceRaw`. It works well.